### PR TITLE
docs: 5.0.0-rc changelog

### PR DESCRIPTION
**Pull request by vchimev**
Add changelog for 5.0.0-rc.

### CODE REVIEW

**Comment by manoldonev** [comment]
Do we have an issue to reference for nested frames feature?

**Comment by manoldonev** [comment]
I updated the "breaking changes" part to reflect the latest state of the support lib story (with restored animators api).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,96 @@
 Cross Platform Modules Changelog
 ==============================
 
+<a name="5.0.0-rc"></a>
+## [5.0.0 - RC](https://github.com/NativeScript/NativeScript/compare/4.2.1...5.0.0-rc) (2018-10-17)
+
+### Bug Fixes
+
+* don't crash on missing resources in tab-view and action-bar ([#6388](https://github.com/NativeScript/NativeScript/issues/6388)) ([56a1b12](https://github.com/NativeScript/NativeScript/commit/56a1b12))
+* include mocha and chai typings ([#6246](https://github.com/NativeScript/NativeScript/issues/6246)) ([9f23be1](https://github.com/NativeScript/NativeScript/commit/9f23be1))
```
Comment (sis0k0) [inline CHANGELOG.md]: This shouldn't be in bug fixes. The packages are added to `devDependencies` - they don't affect the final package.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,96 @@
...
+* nested fragments interact thru child fragment manager ([#6293](https://github.com/NativeScript/NativeScript/issues/6293)) ([3071720](https://github.com/NativeScript/NativeScript/commit/3071720))
```
Comment (sis0k0) [inline CHANGELOG.md]: thru -> through

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,96 @@
...
+* **list-view:** Layout list-view items on request ([#6159](https://github.com/NativeScript/NativeScript/issues/6159)) ([ec24c5a](https://github.com/NativeScript/NativeScript/commit/ec24c5a))
+* **tab-view:** Title and icon positioning ([#6362](https://github.com/NativeScript/NativeScript/issues/6362)) ([e3d5f0d](https://github.com/NativeScript/NativeScript/commit/e3d5f0d))
+* **android:** HEAD request should return statusCode ([7e89f94](https://github.com/NativeScript/NativeScript/commit/7e89f94))
+* **android:** nested frames on app suspend/resume ([#6339](https://github.com/NativeScript/NativeScript/issues/6339)) ([0bf6dc2](https://github.com/NativeScript/NativeScript/commit/0bf6dc2))
+* **android:** parallel navigations should not be triggered ([#6275](https://github.com/NativeScript/NativeScript/issues/6275)) ([6c9fa16](https://github.com/NativeScript/NativeScript/commit/6c9fa16))
+* **android:** second livesync ([#6330](https://github.com/NativeScript/NativeScript/issues/6330)) ([436a318](https://github.com/NativeScript/NativeScript/commit/436a318))
```
Comment (sis0k0) [inline CHANGELOG.md]: I believe that this workaround was removed when we switched back to the animators API.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,96 @@
...
+
+### Features
+
+* add ability to pass touch event through parent view ([#6204](https://github.com/NativeScript/NativeScript/issues/6204)) ([2625683](https://github.com/NativeScript/NativeScript/commit/2625683))
+* implement capitalization type option for prompt dialogs ([#6325](https://github.com/NativeScript/NativeScript/issues/6325)) ([ae6a661](https://github.com/NativeScript/NativeScript/commit/ae6a661))
+* **application-settings:** implemented allKeys method ([#6371](https://github.com/NativeScript/NativeScript/issues/6371)) ([829d18b](https://github.com/NativeScript/NativeScript/commit/829d18b))
+* **image-asset-ios:** add autoScaleFactor option to switch auto scaling ([#6127](https://github.com/NativeScript/NativeScript/issues/6127)) ([81e63ee](https://github.com/NativeScript/NativeScript/commit/81e63ee))
+* **styling:** Added 2 functions to control applicationAdditionalSelectors ([#6124](https://github.com/NativeScript/NativeScript/issues/6124)) ([85b8c01](https://github.com/NativeScript/NativeScript/commit/85b8c01))
```
Comment (sis0k0) [inline CHANGELOG.md]: Added -> add

**Pull request merged**